.travis.yml

```diff
@@ -16,7 +16,7 @@ addons:
 
 before_script:
   - ccache -z
-  - KOKKOS_OPTS=( -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DKokkos_ENABLE_HWLOC=ON )
+  - KOKKOS_OPTS=( -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DKokkos_ENABLE_HWLOC=ON -DKokkos_CXX_STANDARD=14 )
```

Comment:
Do we have to do the same in the jenkins and gitlab-ci file?

Reply:
Don't have those in this repo yet :)